### PR TITLE
Enable support for jemalloc library

### PR DESCRIPTION
**Pull request by eriklindahl**
A number of recent benchmarks indicate that it beats tcmalloc
performance, while avoiding heap bloat issues - and we see
slightly better performance with it.

### CODE REVIEW

File: src/common/CMakeLists.txt [hunk 1]
```diff
@@ -31,6 +31,9 @@ endif()
 if(TCMALLOC_LIBRARY)
   shared_target_link_libraries(mfscommon ${TCMALLOC_LIBRARY})
 endif()
+if(JEALLOC_LIBRARY)
```
Comment (onlyjob) [inline src/common/CMakeLists.txt]: Typo? `JEALLOC` --> `JEMALLOC`?

**Comment by onlyjob** [comment]
Thanks. Did you compare _jemalloc_ performance to default (non-tcmalloc) build?
In the past usage of _tcmalloc_ introduced severe performance regression...

**Comment by onlyjob** [comment]
I'm getting very good performance and RAM usage with _jemalloc_. Based on my positive experience I recommend to merge this patch.

Thanks, @eriklindahl.

File: src/common/CMakeLists.txt (hunk 1 above)
Comment (eriklindahl) [reply src/common/CMakeLists.txt]: Sorry, busy week - this is indeed a mistake!

**Comment by pbeza** [comment]
Thanks for sharing your pull request @eriklindahl. Your pull request probably [will be merged][1] very soon.

[1]: http://cr.skytechnology.pl:8081/#/c/3344/

**Pull request closed without merging**